uploadservice/src/main/java/net/gotev/uploadservice/UploadService.java

```diff
@@ -86,7 +86,7 @@
     /**
      * Buffer size in bytes used for data transfer by the upload tasks.
      */
-    public static int BUFFER_SIZE = 4096;
+    public static int BUFFER_SIZE = 50000;
```

Comment:
The original buffer size was a multiple of 8, which I assume means it deals with numbers of bytes. Do you know whether the size being a multiple of 8 was a significant decision for the library or just an arbitrary choice by the original authors?

Reply:
what about 65536? :)